src/v2/Utils/merchandisingTrial.ts

```diff
@@ -0,0 +1,71 @@
+import { getENV } from "./getENV"
+
+const TARGETED_ARTIST_SLUGS = [
+  "jeff-koons",
+  "marina-abramovic-1",
+  "cindy-sherman",
+  "tracey-emin",
+  "jonas-wood",
+  "andy-warhol",
+  "salvador-dali",
+  "shepard-fairey",
+  "pablo-picasso",
+  "takashi-murakami",
+  "kaws",
+  "alex-katz",
+  "joan-miro",
+  "damien-hirst",
+  "hunt-slonem",
+  "keith-haring",
+  "marc-chagall",
+  "mr-brainwash",
+  "banksy",
+  "henri-matisse",
+  "jim-dine",
+  "david-hockney",
+  "robert-rauschenberg",
+  "sebastiao-salgado",
+  "roy-lichtenstein",
+  "victor-vasarely",
+  "robert-motherwell",
+  "david-shrigley",
+  "julian-opie",
+  "alexander-calder",
+  "daniel-arsham",
+  "david-lachapelle",
+  "ed-ruscha",
+  "ellsworth-kelly",
+  "frank-stella",
+  "gerhard-richter",
+  "hiroshi-sugimoto",
+  "invader",
+  "jean-michel-basquiat",
+  "john-baldessari",
+  "josef-albers",
+  "kenny-scharf",
+  "nobuyoshi-araki",
+  "richard-serra",
+  "robert-longo",
+  "sol-lewitt",
+  "the-connor-brothers",
+  "tom-wesselmann",
+  "vik-muniz",
+  "vivian-maier",
+  "william-kentridge",
+  "yayoi-kusama",
+  "yoshitomo-nara",
+]
+export const TARGETED_ARTIST_ROUTES = TARGETED_ARTIST_SLUGS.map(slug => {
+  return `/artist/${slug}/works-for-sale`
+})
+
+export const getDefaultSortValueByVariant = (artistSlug: string) => {
+  const variant = getENV("ARTIST_GRID_MANUAL_CURATION_TRIAL_QA")
+  const isTargetedArtistSlug = TARGETED_ARTIST_SLUGS.includes(artistSlug)
+
+  if (isTargetedArtistSlug && variant === "control") {
```

Comment:
Ah good q! This does look fishy, but in this experiment:

- the control group = let default sort do its thing, **except for** pinning artworks to featured slots, so they see an _uncurated_ grid

- the experiment group = default sort with the usual pinning of artworks works, so they see a curated grid.

The experiment here really is: what is the measurable effect of doing of lot of manual curation?

@dimatretyak feel free to chime in if I've misstated anything.

Reply:
Yep, it's a bit unusual but Roop's breakdown is accurate.

We want to test if artist artworks grids which are manually curated perform better than our default sort algorithm. Since we don't want to interfere with artist pages outside of the scope of this A/B test, we decided to define a new sort option to use for the control, and let the experiment trigger normal behavior including currently enabled curation, knowing that we'll also be seeding those `Artwork#featured_slot` values separately.

The A/B test is web only but these curations which also trickle down to the mobile apps during the test.

Reply:
@anandaroop you have described everything correctly. Thank you for it!